.github/snippet-bot.yml

```
@@ -0,0 +1,3 @@
+ignoreFiles:
+  - test/fixtures/**/*
+  - test/fixtures/**/*
```

Did you mean to use this instead?
```suggestion
  - test/specs/lib/**/*
```

---

Seems like there's no region tags added from `specs/lib` but was wondering since you had two lines instead of just one

---

Oops, left an extra line (I had one for capturing .ts files and .js files)! I think we might as well cover the files in your suggestion too!